kvdb/postgres/db_test.go

```diff
@@ -0,0 +1,199 @@
+package postgres
+
+import (
+	"testing"
+
+	"github.com/btcsuite/btcwallet/walletdb"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/net/context"
+)
+
+func TestDb(t *testing.T) {
+	const (
+		dsn    = "postgres://bottle:bottle@localhost:45432/lnd?sslmode=disable"
```

Comment:
Will this test run without the user running the DB first?

Reply:
No, can't say this is a unit test. Still need to decide what to do with it.

Reply:
Fixed this using embedded postgres